src/components/ProfileInformation/ProfileInformation.tsx

```diff
@@ -0,0 +1,49 @@
+import { Icon } from 'semantic-ui-react'
```

Comment:
Would you mind changing the way we import this component? The idea is to import only the Icon.
```suggestion
import Icon from 'semantic-ui-react/dist/commonjs/elements/Icon/Icon'
```